styles/styles.css

```diff
@@ -217,9 +217,113 @@ main .section.highlight {
   color: var(--midnight-blue-color);
 }
 
+.default-content-wrapper h2 {
+  margin-top: 60px;
+}
+
+.default-content-wrapper h3:nth-child(odd) {
+  float: right;
+  width: 50%;
+  padding-left: 50%;
+  font-size: var(--paragraph-font-size);
+  margin-top: 80px;
+}
+
+.default-content-wrapper h3:nth-child(odd) + p {
+  float: left;
+  width: 50%;
+  margin-top: -30px;
+  height: 306px;
+}
+
+.default-content-wrapper h3:nth-child(odd) + p img{
+  height: 100%;
+}
+
+.default-content-wrapper h3:nth-child(even) {
+  float: left;
+  width: 50%;
+  padding-right: 50%;
+  font-size: var(--paragraph-font-size);
+  margin-top: 80px;
+}
+
+.default-content-wrapper h3:nth-child(even) + p {
+  float: right;
+  width: 50%;
+  margin-top: -30px;
+  height: 306px;
+}
+
+
+.default-content-wrapper h3:nth-child(even) + p img{
+  height: 100%;
+  float: right;
+}
+
+.default-content-wrapper h3:nth-child(odd) + p + p {
+  padding-left: 50%;
+  line-height: 27px;
+}
+
+.default-content-wrapper h3:nth-child(even) + p + p {
+  padding-right: 50%;
+  line-height: 27px;
+}
+
+.default-content-wrapper .info-container {
+  position: absolute;
+  top: 550px;
+  right: 130px;
+}
+
+.default-content-wrapper .info-container .info-number {
+  font-size: 50px;
+  line-height: 70px;
+  font-weight: 300;
+}
+
+.default-content-wrapper .info-container .info-text {
+  font-size: 15px;
+  line-height: 24px;
+  padding-left: 5px;
+  vertical-align: super;
+}
+
+.default-content-wrapper .info {
+  background-color: var(--background-color);
+  opacity: 0.9;
+  padding: 10px 20px;
+  margin-bottom: 20px;
+}
+
+.default-content-wrapper .spec-container {
+  margin-top: 86px;
+}
 
+/* stylelint-disable-next-line no-descending-specificity */
+.default-content-wrapper .spec-container p {
+  margin-left: 0 !important;
```

Comment:
do we really need important?

Reply:
Yes because it is been overridden by other style`body.ship-focus .default-content-wrapper p:not(p ~ p) `.

Reply:
Removed